shuffle-tools/1.2.0/src/app.py

```diff
@@ -2633,14 +2633,10 @@ def run_ssh_command(self, host, port, user_name, private_key_file_id, password,
         try:
             stdin, stdout, stderr = ssh_client.exec_command(str(command))
 
-            stderr_ouput = stderr.read().decode(errors='ignore')
-
-            if stderr_ouput:
-                return {"success": "true", "message": stderr_ouput}
         except Exception as e:
             return {"success":"false","message":str(e)}
 
-        return {"success":"true","output": stdout.read().decode(errors='ignore')}
+        return {"success":"true","output": stdout.read().decode(errors='ignore'), "error_log": stderr.read().decode(errors='ignore')}
```

Comment:
Shouldn't stderr.read() and stdout.read() be try-caught? Those can easily crash. The decode as well.